### ISSUE TITLE
GSL error when reconstructing with Monopod (Trac #1849)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1841]
I'm getting GSL errors when trying to reconstruct HESE Hitspool events using Monopod. A typical command would be:
python /home/hitspool/HsHeseProcess/MonopodReco.py -i /data/user/hitspool/HESE_HS/HESE_20160528_202307/I3Files/HESE_20160528_202307_h
itspool_eventframe.i3.gz -o /data/user/asteuer/atest.i3

which would produce the error:
gsl: brent.c:57: ERROR: function value is not finite
Default GSL error handler invoked.
Aborted

Migrated from https://code.icecube.wisc.edu/ticket/1849
```json
{
    "status": "closed", 
    "changetime": "2016-09-01T18:06:50", 
    "description": "I'm getting GSL errors when trying to reconstruct HESE Hitspool events using Monopod. A typical command would be:\npython /home/hitspool/HsHeseProcess/MonopodReco.py -i /data/user/hitspool/HESE_HS/HESE_20160528_202307/I3Files/HESE_20160528_202307_h\nitspool_eventframe.i3.gz -o /data/user/asteuer/atest.i3\n\nwhich would produce the error:\ngsl: brent.c:57: ERROR: function value is not finite\nDefault GSL error handler invoked.\nAborted\n", 
    "reporter": "asteuer", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1472753210982102", 
    "component": "combo reconstruction", 
    "summary": "GSL error when reconstructing with Monopod", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-09-01T16:35:45", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160901T16:45:32: 
* jvansanten commented:

> With assertions on, this trips:
> 
> ```text
> MillipedeDOMCacheMap.cxx:148: int rebin_min_entries(int, const double*, const double*, bool*, double*, double*, double, double): Assertion `bin_edges[q-1] < raw_bin_edges[raw_bins]' failed.
> ```
> This likely makes the noise contribution negative, leading to the GSL assertion above.


* jvansanten changed owner from "" to "jvansanten"
* jvansanten changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160901T18:06:50: 
* jvansanten commented:

> In changeset 149760:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="149760"
> Raise an error if a pulse starts _exactly_ at the end of the readout
> window. This results in zero-width bins, and is likely a sign that
> readout window was chosen from the times of the observed pulses, biasing
> the noise estimate. Closes #1841.
> ```


* jvansanten changed resolution from "" to "fixed"
* jvansanten changed status from "accepted" to "closed"
